ipatests/test_integration/test_external_ca.py

```diff
@@ -195,6 +195,15 @@ def test_switch_to_external_ca(self):
         result = check_CA_flag(self.master)
         assert bool(result), ('External CA does not have "C" flag')
 
+    def test_issuerDN_after_renew_to_external(self):
+        """ Check if issuer DN is updated after self-signed > external-ca
+
+        This test checks if issuer DN is updated properly after CA is
+        renewed from self-signed to external-ca
+        """
+        result = self.master.run_command(['ipa', 'ca-show', 'ipa'])
+        assert "Issuer DN: CN=example.test" in result.stdout_text
```

Comment:
Hi @mrizwan93 
Thanks for the test, looks good to me. I have just a nitpick, the Issuer DN value for the test is hardcoded and used in multiple places, you may want to define a constant in ipatests/create_external_ca.py and use this constant everywhere.

Reply:
Hello @flo-renaud ,
I am seeing this hardcoded value only at `def create_ca(self, cn='example.test')` in `ipatests/create_external_ca.py`. So I guess it is not worth to have constant for it.

or am I missing something?

Reply:
Hi @mrizwan93 
the hardcoded value can be found:

-  in ipatests/test_integration/test_external_ca.py:
```
def check_CA_flag(host, nssdb=paths.PKI_TOMCAT_ALIAS_DIR,
                  cn='example.test'):
```
- in ipatests/test_integration/test_external_ca.py: in test_issuerDN_after_renew_to_external:
```
assert "Issuer DN: CN=example.test" in result.stdout_text
```
- in ipatests/create_external_ca.py:
```
def create_ca(self, cn='example.test'):
```

If the default value in create_ca ever gets modified, it will be hard to remember to modify the value in the other places.